src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBiddingClosedMessage.tsx

```diff
@@ -26,7 +27,10 @@ const BiddingClosedMessage: React.FC<BiddingClosedMessageProps> = ({
             {t(`artworkPage.sidebar.createAlert.description`)}
           </Text>
           <Spacer y={2} />
-          <ArtworkSidebarCreateAlertButtonFragmentContainer artwork={artwork} />
+          <ArtworkCreateAlertButtonFragmentContainer
+            artwork={artwork}
+            analytics={ContextModule.artworkSidebar}
```

Comment:
**suggestion**: I think we should either rename this prop so that it reflects a single named attribute or change the prop type to an object which we can fold into the properties sent under the hood:

**Option 1**

```tsx
<ArtworkCreateAlertButtonFragmentContainer
  analyticsContextModue={ContextModule.artworkSidebar}
/>
```

```typescript
analytics: {
  contextModule: props.analyticsContextModule,
  intent: Intent.createAlert,
}
```

**Option 2**

```tsx
<ArtworkCreateAlertButtonFragmentContainer
  analytics={{
    contextModule: ContextModule.artworkSidebar
  }}
/>
```

```typescript
analytics: {
  intent: Intent.createAlert,
  ...props.analytics
}
```